src/rgw/rgw_rest_info.h

```diff
@@ -0,0 +1,33 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab ft=cpp
+
+#pragma once
+
+#include "rgw_rest.h"
+#include "rgw_rest_s3.h"
+
+
+class RGWHandler_Info : public RGWHandler_Auth_S3 {
+protected:
+  RGWOp *op_get() override;
+public:
+  using RGWHandler_Auth_S3::RGWHandler_Auth_S3;
+  ~RGWHandler_Info() override = default;
+
+  int read_permissions(RGWOp*, optional_yield) override {
+    return 0;
+  }
+};
+
+class RGWRESTMgr_Info : public RGWRESTMgr {
```

Comment:
in rgw_main.cc, we need to register this under the admin mgr:
```c++
admin_resource->register_resource("info", new RGWRESTMgr_Info);
```
that registers it for requests like "GET /admin/info"

Reply:
will fix